### ISSUE TITLE
Questions related to score after predictions

### ISSUE THREAD

**Issue by vsay01** [pytorch/android-demo-app#60]
From each prediction, we get back score of each result. I know that the number that close to 0 is the best predicted result.

My questions are 

1. How is the score calculated internally ?
2. Why the score is negative?
3. How to get percentage of the result based on the score?

Thanks.

**Comment by ptrblck** [comment]
Also asked [here](https://discuss.pytorch.org/t/pytorch-android-score/69794/2?u=ptrblck).
Lets keep the discussion in one place please. :)

**Comment by vsay01** [comment]
@ptrblck thanks.

I added:

# Add softmax layer
mobilev2_loaded_model = nn.Sequential(
    mobilev2_loaded_model,
    nn.Softmax(1)
)

and serialized model again after that in mobile I can multiple 100 to the score and get the percentage of that class.
